jobs/future-effective-filings/file_future_effective.py

```diff
@@ -125,7 +125,7 @@ async def run(loop, application: Flask = None):
             for filing in filings:
                 filing_id = filing['filing']['header']['filingId']
                 effective_date = filing['filing']['header']['effectiveDate']
-                # TODO Use UTC time?
+                # NB: effective_date and now are both UTC
```

Comment:
should this just be deleted?

Reply:
I thought the comment had value but I'm OK with just deleting it.

Do any other reviewers have a strong opinion about this?

Reply:
Either is fine with me :)